tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -7,12 +7,22 @@ class BugzillaLite(Base):
     _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
     _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
     _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _profile_icon = (By.CSS_SELECTOR, "a[href='/profile/']")
+    _button_logout = (By.CSS_SELECTOR, "a[href='/logout/']")
     _dashboard_navigator_locator = (By.ID, 'dashboardNav')
     _dashboard_login_locator = (By.ID, 'login')
     _popup_intro = (By.ID, 'intro')
     _button_popup_intro = (By.ID, 'intro-submit')
-
-    def login(self, username, password):
+    _assigned_icon = (By.CSS_SELECTOR, '#dashboardNav .assigned')
+    _flagged_icon = (By.CSS_SELECTOR, '#dashboardNav .flagged')
+    _filed_icon = (By.CSS_SELECTOR, '#dashboardNav .filed')
+    _search_bar = (By.ID, 'searchLink')
+    _button_cancel_search = (By.LINK_TEXT, 'Cancel')
```

Comment:
This selector will break the day where bzlite is translated. I propose: `By.CSS_SELECTOR, '#search a[href="/"']`